src/Storage/Storage/File/Cmdlet/SetAzureStorageFileContent.cs

```diff
@@ -78,16 +64,6 @@ public class SetAzureStorageFileContent : StorageFileDataManagementCmdletBase, I
             ParameterSetName = LocalConstants.DirectoryParameterSetName,
             HelpMessage = "CloudFileDirectory object indicated the cloud directory where the file would be uploaded.")]
         [ValidateNotNull]
-        [Alias("CloudFileDirectory")]
-        public CloudFileDirectory Directory { get; set; }
-
-        [Parameter(
-            Mandatory = false,
-            ValueFromPipeline = true,
-            ValueFromPipelineByPropertyName = true,
-            ParameterSetName = LocalConstants.DirectoryParameterSetName,
-            HelpMessage = "CloudFileDirectory object indicated the cloud directory where the file would be uploaded.")]
-        [ValidateNotNull]
         public ShareDirectoryClient ShareDirectoryClient { get; set; }
```

Comment:
the HelpMessage of the parameter is not correct.
It's "ShareDirectoryClient object", not "CloudFileDirectory object".

There should be multiple place need this kind of update in this PR.

Reply:
Updated in all possible places and corresponding help files